config.toml

```diff
@@ -1,10 +1,8 @@
 baseURL = "http://startwords.cdh.princeton.edu/"
-languageCode = "en-us"
```

Comment:
Any reason this is removed?

Reply:
I couldn't imagine a scenario where the site's language code would be anything other than `en-us`; even if we were to publish a full article in another language we would need to scope a `lang=` attribute to the `article` in question. it didn't seem like something worth storing in a config.